### ISSUE TITLE
`no-incorrect-template-string-interpolation` can conflict with `prefer-unicode-code-point-escapes`

### ISSUE THREAD

**Issue by alfaproject** [sindresorhus/eslint-plugin-unicorn#3511]
We get a report here from `prefer-unicode-code-point-escapes`
```js
    const blob = new Blob([`\uFEFF${csv}`], { type: 'text/csv;charset=utf-8;' });
```

So after fixing we get this:
```js
    const blob = new Blob([`\u{FEFF}${csv}`], { type: 'text/csv;charset=utf-8;' });
```

Which then triggers `no-incorrect-template-string-interpolation`
